llvm/lib/Target/Mips/MipsISelLowering.cpp

```diff
@@ -3381,7 +3381,7 @@ MipsTargetLowering::LowerCall(TargetLowering::CallLoweringInfo &CLI,
 
       // Collect CSInfo about which register passes which parameter.
       const TargetOptions &Options = DAG.getTarget().Options;
-      if (Options.SupportsDebugEntryValues)
+      if (Options.EmitCallSiteInfo && Options.SupportsDebugEntryValues)
```

Comment:
Actually, looking at other platforms, it seems like it’s more common to only do this when EmitCallSiteInfo is true. A comment on the previous phabricator patch mentions this. 

If MIPS requires CallSiteInfo for debugging for some reason, this can be an OR expression. 

Can you update the patch to match the other platforms here?

Reply:
Let's ask @djolertrk.

https://github.com/llvm/llvm-project/commit/170ac4be3392201d5f5e124e8a1b7d78de3f82c8
https://reviews.llvm.org/D78105

Reply:
Yeah, we should check for `EmitCallSiteInfo` only here. The Debug Entry Values Feature relies on this info, but the call site information could be used for some other things as well.

Reply:
Updated the patch as suggested! Thank you.

Reply:
LGTM. I will accept this PR after CI finish successfully.